client/state/sites/selectors.js

```diff
@@ -244,7 +246,7 @@ export function getSiteSlug( state, siteId ) {
 	}
 
 	return urlToSlug( site.URL );
-}
+}, getSitesItems );
```

Comment:
The `getDependants` function is supposed to return an array, so it's a bit better to write `[ getSitesItems ]` here. The `createSelector` function does this conversion to one-element array automatically. But if the `getSitesItems` selector started returning arrays, it would make the selector buggy.